docs/getting-started.md

```diff
@@ -232,7 +224,682 @@ curl localhost:10101/index/repository/query \
 Please note that while user ID 99999 may not be sequential with the other column IDs, it is still a relatively low number. 
 Don't try to use arbitrary 64-bit integers as column or row IDs in Pilosa - this will lead to problems such as poor performance and out of memory errors.
 
+#### Using Go
+
+Pilosa requires Go 1.12 or higher. It is also recommended that you have a code editor downloaded.
+
+##### Create the Environment
+
+In order to communicate with Pilosa through your Go code, you must have a "translator," which is go-pilosa. To install go-pilosa, open a terminal (one other than the one running Pilosa) and download the library in your `GOPATH` using:
```

Comment:
We don't normally use the term "translator" for this, and it does have another meaning in the context of Pilosa - "client" would be a better term.